Documentation/microsoft-git.md

```diff
@@ -0,0 +1,39 @@
+Details about the Microsoft/Git Fork
+====================================
+
+The [`microsoft/git` fork of Git](https://github.com/microsoft/git)
+is used to take early performance patches or custom patches for
+[VFS for Git](https://github.com/microsoft/git).
+
+Integration Testing with C# Layer
+---------------------------------
+
+If you have a pull request in `microsoft/git`, then you
+can test integration in VFS for Git using a pull request
+there.
+
```

Comment:
First, build a custom Git installer for your PR merge commit.  The GVFS testsuite will install this on the test machine and use it to support the GVFS functional tests.   This build step it in a different repo that is unknown to `microsoft/git`.   Run the ...